adapters/tlsio_wolfssl.c

```diff
@@ -562,6 +565,14 @@ static int create_wolfssl_instance(TLS_IO_INSTANCE* tls_io_instance)
         wolfSSL_SetIOWriteCtx(tls_io_instance->ssl, tls_io_instance);
         wolfSSL_SetIOReadCtx(tls_io_instance->ssl, tls_io_instance);
 
+#ifdef HAVE_SECURE_RENEGOTIATION
+        if(wolfSSL_UseSecureRenegotiation(tls_io_instance->ssl) != SSL_SUCCESS)
+        {
+            LogError("unable to enable secure renegotiation");
+            return MU_FAILURE;
```

Comment:
I believe the code style in this repo is to have one return point except parameter checking. This should likely be
```suggestion
            result = MU_FAILURE;
```
and may have to move around to work with the other `if/else if's`

Reply:
In this case, if result = MU_FAILURE, result will get reset to 0 at line 577.  So it should be returned here.